react/Radio/index.jsx

```diff
@@ -18,6 +25,9 @@ const Radio = props => {
     checked: checkedProp,
     ...restProps
   } = props
+  logRadioDepecrated(
+    'The Radio component is deprecated, please use `<Radios />` instead. You can simply replace `<Radio value="" label="" />` by `<FormControlLabel value="" label="" control={<Radios />} />`. See documentation for more details.'
```

Comment:
we avoid the use of `simply` or `easily` or else. It's never as simple or easy as it seems